.github/workflows/sbt-node-snyk.yml

```diff
@@ -0,0 +1,37 @@
+name: SBT Node Snyk
+
+on:
+  workflow_call:
```

Comment:
@jfsoul, this is a good point. Our workflows diverged from that recommendation due to failing checks against PRs which were nothing to do with the content of the PR. As many of our projects have outstanding vulnerabilities which need resolving, the test function would always provide a failing result. This was unhelpful and indicated an issue to developers unfamiliar with this process. As it represented a problem with overall codebase and not the PR specifically we felt it wasn't relevant to have as a PR check. 

We therefore thought it would be preferable to exclude the test check and only run monitor, the results of which can be reviewed in the Snyk dashboard. It would be interesting to know how other teams addressed these concerns.

Reply:
After discussing offline I realise now that workflow-frontend will actually only be running this workflow on pushes to main - we've just commented that restriction out for testing. I do think there's value in `snyk test` for at least some teams, but I think this is an area that needs more thought before we can form a solid recommendation.